nselib/http.lua

```diff
@@ -1640,6 +1644,30 @@ function get(host, port, path, options)
     response, state = lookup_cache("GET", u.host, u.port, u.path, options);
     if ( response == nil ) then
       response = generic_request(u.host, u.port, "GET", u.path, options)
+      if(response.cookies and #response.cookies>0) then 
+        if (options and options.cookies and #options.cookies>0) then
+          --We replace the value of the cookie if same name cookie exists
+          --Else, we append it in the end.
+          local flag = 0
+          for k,v in pairs(response.cookies) do
+            for k1,v1 in pairs(options.cookies) do
+              flag = 0   
+              if(v.name == v1.name) then
+                options.cookies[k1].value = response.cookies[k].value
+                flag = 1
```

Comment:
Also remember that we have stdnse.contains() to check if an entry exists on a table.